ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,209 @@
+---
+eip: xxxx
+title: Compressed Display Format for EVM Addresses
+description: A display format for abbreviating consecutive identical hex characters in EVM addresses using run-length encoding for UI and logging.
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-compressed-display-format-for-evm-addresses/27360
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-8117-compressed-display-format-for-evm-addresses/27360
```
Updated with assigned number.

Recommend that the discussions topic isn't a complete copy of the ERC text as this will likely diverge.  Recommend just having a link and perhaps the abstract